config/sort2.h

```diff
@@ -64,6 +66,7 @@ enum SortType
   SORT_PATH     = 18, ///< Sort by the folder's path
   SORT_LABEL    = 19, ///< Sort by the emails label
   SORT_DESC     = 20, ///< Sort by the folder's description
+  SORT_UNSORT   = 21, ///< Sort by the original configuration order
```

Comment:
Ah, sorry.  I should have been clearer.
We don't need a `SORT_UNSORT`.
Traditionally `SORT_ORDER` has meant -- 'in config file order'.
I've made a note to review the naming, to try to make it more consistent.

Reply:
It's ok, I could've been more perceptive :-)
Fixed.